applications/matter_bridge/doc/adding_bridged_protocol.rst

```diff
@@ -0,0 +1,20 @@
+.. _matter_adding_bridged_protocol:
+
+Adding support for bridging the Matter network to a proprietary protocol
+************************************************************************
+
+The :ref:`Matter Bridge architecture <ug_matter_overview_bridge_ncs_implementation>` describes two sides of bridging relationship, as ``Matter Bridged Device`` and ``Bridged Device Data Provider``.
+The ``Matter Bridged Device`` represents the side that natively supports Matter, and the ``Bridged Device Data Provider`` represents non-Matter interface.
+It is not specified what protocol should be used for a ``Bridged Device Data Provider`` role.
+
+The Matter Bridge application supports bridging Matter to Bluetooth LE data providers that use Bluetooth LE protocol to communicate with physical end devices and simulated data providers that emulate real devices behavior in the application.
+You can add support for any protocol you want, as long as you are able to describe data models translation between Matter and that protocol.
+
+To provide support for a new protocol, you have to implement the ``Bridged Device Data Provider`` role that is able to communicate with the physical end devices using that protocol.
+The interactions to be implemented by the ``Bridged Device Data Provider`` vary depending on the bridged Matter device type.
+
+Typically the ``Bridged Device Data Provider`` has to provide up to date information about the device's state by pushing notification about the state changes.
+It allows to update the Matter Data Model state and also enables handling Matter subscriptions.
+The subscription handling can be implemented either by utilizing protocol native support for pushing notifications by the end device or performing periodic data polling.
```

Comment:
```suggestion
The subscription handling can be implemented either by utilizing protocol native support for pushing notifications by the end device or by performing periodic data polling.
```
Not sure if it should be "protocol-native support" (describing what type of support is talked about) or "the implemented protocol's native support", or perhaps even "Matter's native support".

Reply:
It is related to the protocol that is used in bridge alongside Matter, so I used the second suggestion.